### PR TITLE
Update messages_sk.yml

### PR DESCRIPTION
**Pull request by Thymue**
Hi, i translated what i could but some translations can't be translated due to how Slovak language works. :D
example:
I'll be back in five minutes.
Budem späť za päť minút.
I'll be back in four minutes.
Budem späť za štyri minúty.
Basically it goes
one minute, two minutes, three minutes, four minutes, five minutes, six minutes
jedna minúta, dve minúty, tri minúty, štyri minúty, päť minút, šesť minút and i think it goes to infinity with "minút"
(i will comit again for translating capatcha because i need to see it in game)

### CODE REVIEW

**Comment by ljacqu** [comment]
> and what about contributors credits ? -Thymue

Feel free to add your name to the header. We'll also add you to https://github.com/AuthMe/AuthMeReloaded/wiki/Translators

> minúta, minúty and minút

We will address this with #1159. For now we need to either use one form or to use abbreviations (e.g. min.)

**Comment by Thymue** [comment]
I'm a little bit confused from GitHub but i think i commited another update
If not i have it on my PC

**Comment by ljacqu** [comment]
@Thymue It's strange, I see your commit on GitHub but this pull request somehow wasn't updated. In https://github.com/Thymue/AuthMeReloaded/commit/fe99555d2f0d5859037a148908272983db87ffc9, I see you translated `<theCaptcha>` in `usage_captcha` and `THE_CAPTCHA` in `wrong_captcha`—they are, however, placeholders that will be replaced. Could you please add them back? Maybe with that change the pull request will also be updated again.

**Comment by Thymue** [comment]
Ah..
you didn't read my comment
"Okay now can you tell me how/where is "# TODO: Missing tags %reg_count, %max_acc, %reg_names" used ?
send me screenshot or something if you can :)"

**Comment by ljacqu** [comment]
that message is shown when a IP address has already registered as many users as it may. %reg_count is how many accounts are registered to the IP, %max_acc is the configured max and %reg_names is the list of names that has been registered by the IP addr

**Comment by Thymue** [comment]
Ooh... it's missing tags for max_reg it's not another message to be translated,
It makes a lot more sense now 😃.

File: src/main/resources/messages/messages_sk.yml [hunk 1]
```diff
@@ -21,20 +22,19 @@ password_error_chars: '&4Tvoje heslo obsahuje zakázané znaky. Povolené znaky:
...
 
 # Errors
 unknown_user: '&cZadané meno nie je zaregistrované!'
 denied_command: '&cPre použitie tohto príkazu sa musíš prihlásiť!'
 denied_chat: '&cMusíš byť prihlásený ak chceš použiť chat!'
 not_logged_in: '&cNie si ešte prihlásený!'
 tempban_max_logins: '&cBol si dočasne zabanovaný za opakované zadanie zlého hesla.'
-# TODO: Missing tags %reg_count, %max_acc, %reg_names
-max_reg: '&fDosiahol si maximum registrovaných účtov.'
+max_reg: '&fPrekročil si maximum registrovaných účtov(%reg_count/%max_acc|%reg_names).'
```
Comment (ljacqu) [inline src/main/resources/messages/messages_sk.yml]: Maybe needs a space between `účtov` and the bracket?

**Comment by ljacqu** [comment]
Hi @Thymue , thank you so much for your work! After we got the SK translation I don't think we've ever received any updates for it, so it was desperately old! Is it OK to merge now?

**Comment by Thymue** [comment]
I'll do one last change and then you can.
And I need to thank you and the AuthMe team for keeping this plugin free.. 🙂

**Comment by ljacqu** [comment]
Excellent, thank you very much @Thymue :)

**Comment by Thymue** [comment]
I know I'm annoying but haven't you forget to change it in docs?

**Comment by ljacqu** [comment]
You're right, and you're not annoying—it's great to see people that think along! :smile:
To avoid "commit spam" we don't update the translations page after each change, but you're right that there have been considerable changes (not only for SK) since the last update, so let's push one. Thanks for the heads up

**Pull request merged**